CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.3] 2020-02-11
```

Comment:
You also need to update the links section at the end of this file (next to `[0.5.2]`), to add links for the upcoming release.

Reply:
:+1:

Reply:
Check https://github.com/giantswarm/architect-orb/pull/62/files